packages/nimble-blazor/SprightBlazor/SprightBlazor.csproj

```
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <DebugType>embedded</DebugType>
+    <Authors>NI</Authors>
+    <PackageProjectUrl>https://github.com/ni/nimble</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/ni/nimble</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
+    <PackageTags>Web Components, .NET, Spright, Nimble, FAST, Blazor, .NET 6.0</PackageTags>
+    <NeutralLanguage>en</NeutralLanguage>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702,8669,1591</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <NoWarn>CS8669</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Remove="package.json" />
+    <Content Remove="packages.lock.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="CodeAnalysisDictionary.xml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="..\CodeAnalysisDictionary.xml" Link="CodeAnalysisDictionary.xml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <SupportedPlatform Include="browser" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="6.0.28" />
+    <PackageReference Include="NI.CSharp.Analyzers" Version="[2.0.21]" PrivateAssets="all" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="wwwroot\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="package.json" />
+    <None Include="packages.lock.json" />
+  </ItemGroup>
+
+  <Target Name="CopyResources" BeforeTargets="BeforeBuild">
+    <Exec Command="cd ../ &amp;&amp; npm run copy-resources:spright" />
```

There is a [WorkingDirectory option](https://learn.microsoft.com/en-us/visualstudio/msbuild/exec-task?view=vs-2022#parameters), something like

```suggestion
    <Exec Command="npm run copy-resources:nimble" WorkingDirectory="../" />
```